pkg/hubble/metrics/api/context.go

```diff
@@ -352,6 +349,13 @@ func handleReservedIdentityLabels(lbls []string) string {
 	return ""
 }
 
+func destinationNamespaceContext(flow *pb.Flow) (context string) {
+	if flow.GetDestination() != nil {
+		context = flow.GetDestination().Namespace
+	}
+	return
+}
```

Comment:
this func only moved right? Any reason behind re-ordering?

Reply:
I think this only moved in the interim commit, it was just weird that this wasn't grouped with the other destination* options. In a later commit I refactored all this so we don't even have this function anymore. https://github.com/cilium/cilium/pull/21181/commits/bd7b28723bc995c87411ef9b2ba93a03c109d01d